# HG changeset patch
# User Lachlan Andrew <lachlanbis@gmail.com>
# Date 1468397831 -36000
#      Wed Jul 13 18:17:11 2016 +1000
# Node ID bb4a05cb4a0257b9e1cf3e3ac368d072bd5cd4d3
# Parent  879f297f31945a259f30f1a0032a13f42320880e
Change Ctrl+PgDown to move the focus rightward, not PgUp (bug #48495).

* shortcut-manager.cc (shortcut_manager::do_init_data): Swap PageUp with
PageDown.

diff --git a/libgui/src/shortcut-manager.cc b/libgui/src/shortcut-manager.cc
--- a/libgui/src/shortcut-manager.cc
+++ b/libgui/src/shortcut-manager.cc
@@ -318,23 +318,23 @@ shortcut_manager::do_init_data ()
   // help
   init (tr ("Help on Keyword"), "editor_help:help_keyword",
         QKeySequence::HelpContents);
   init (tr ("Document on Keyword"), "editor_help:doc_keyword",
         QKeySequence (Qt::SHIFT + Qt::Key_F1));
 
   // tab navigation
   init (tr ("Switch to Left Tab"), "editor_tabs:switch_left_tab",
-        QKeySequence (ctrl + Qt::Key_PageDown));
+        QKeySequence (ctrl + Qt::Key_PageUp));
   init (tr ("Switch to Right Tab"), "editor_tabs:switch_right_tab",
-        QKeySequence (ctrl + Qt::Key_PageUp));
+        QKeySequence (ctrl + Qt::Key_PageDown));
   init (tr ("Move Tab Left"), "editor_tabs:move_tab_left",
-        QKeySequence (Qt::AltModifier + Qt::Key_PageDown));
+        QKeySequence (Qt::AltModifier + Qt::Key_PageUp));
   init (tr ("Move Tab Right"), "editor_tabs:move_tab_right",
-        QKeySequence (Qt::AltModifier + Qt::Key_PageUp));
+        QKeySequence (Qt::AltModifier + Qt::Key_PageDown));
 
 }
 
 void
 shortcut_manager::init (QString description, QString key, QKeySequence def_sc)
 {
   QKeySequence actual
     = QKeySequence (_settings->value ("shortcuts/"+key, def_sc).toString ());
